make/integration-tests.mk

```diff
@@ -93,4 +101,7 @@ test-integration: clean-cov generate fmt vet ginkgo ## Requires kubernetes clust
 		--fail-on-pending \
 		--keep-going \
 		--trace \
+		--race \
+		--output-interceptor-mode=none \
+		-v \
```

Comment:
what about adding `VERBOSE` makefile variable to opt-in (by default false) ? Same as unit tests.